x-pack/plugins/observability_solution/observability_ai_assistant_app/server/functions/query/validate_esql_query.ts

```diff
@@ -8,11 +8,11 @@
 import { validateQuery } from '@kbn/esql-validation-autocomplete';
 import { getAstAndSyntaxErrors } from '@kbn/esql-ast';
 import type { ElasticsearchClient } from '@kbn/core/server';
-import { ESQLSearchReponse } from '@kbn/es-types';
+import { ESQLSearchReponse, ESQLRow } from '@kbn/es-types';
 import { esFieldTypeToKibanaFieldType, type KBN_FIELD_TYPES } from '@kbn/field-types';
 import { splitIntoCommands } from './correct_common_esql_mistakes';
 
-export async function validateEsqlQuery({
+export async function runAndValidateEsqlQuery({
```

Comment:
If you're running the query anyways, is there any point in also performing the client-side validation? You'll already get the server-side errors...

Reply:
For better user messages to the LLM, we have some improvements on our side so it was mostly for this. (and the fact that we can add more things on our side in the future)

Reply:
Ok makes sense.